prepare-vms/scripts/postprep.rc

```diff
@@ -31,22 +31,29 @@ STEP = 0
 START = time.time()
 
 def bold(msg):
-    return "{} {} {}".format("$(tput smso)", msg, "$(tput rmso)")
+    return "{} {} {}".format("", msg, "")
 
-def system(cmd):
+def system(cmd, ok_codes=[]):
     global STEP
     with open("/tmp/pp.status", "a") as f:
         t1 = time.time()
         f.write(bold("--- RUNNING [step {}] ---> {}...".format(STEP, cmd)))
         retcode = os.system(cmd)
-        if retcode:
-            retcode = bold(retcode)
         t2 = time.time()
         td = str(t2-t1)[:5]
-        f.write("[{}] in {}s\n".format(retcode, td))
+        f.write("[{}] in {}s\n".format(bold(retcode) if retcode else retcode, td))
         STEP += 1
     with open("/home/ubuntu/.bash_history", "a") as f:
         f.write("{}\n".format(cmd))
+    ret = {"retcode": retcode}
```

Comment:
Why using this `ret` variable? It seems like it's only used in `return` so I wonder if we could just `return dict(retcode=retcode, cmd=cmd)` ?